script.js

```diff
@@ -1,4 +1,118 @@
+var win_count = 0;
+var loss_count = 0;
+var name_input = "";
+
+var random_pss = function () {
+  var paper = "Paper";
+  var scissors = "Scissors";
+  var stone = "Stone";
+  //random numbers from 1 - 3
+  var choices = Math.floor(Math.random() * 3) + 1;
+  // randomize the paper scissors stone
+  if (choices == 1) {
+    return paper;
+  } else if (choices == 2) {
+    return scissors;
+  } else if (choices == 3) {
+    return stone;
+  }
+};
+
+var process = function (mypov, computer_pov) {
+  //normal stream
+
+  if (mypov == "Scissors" && computer_pov == "Paper") {
+    win_count += 1;
+    return "you win!";
+  } else if (mypov == "Scissors" && computer_pov == "Stone") {
+    loss_count += 1;
+    return "you lose!";
+  } else if (mypov == "Paper" && computer_pov == "Stone") {
+    win_count += 1;
+    return "you win!";
+  } else if (mypov == "Paper" && computer_pov == "Scissors") {
+    loss_count += 1;
+    return "you lose!";
+  } else if (mypov == "Stone" && computer_pov == "Scissors") {
+    win_count += 1;
+    return "you win!";
+  } else if (mypov == "Stone" && computer_pov == "Paper") {
+    loss_count += 1;
+    return "you lose!";
+  }
+
+  //reversed stream
+  if (mypov == "Reversed Scissors" && computer_pov == "Paper")
+    return "you lose!";
+  else if (mypov == "Reversed Scissors" && computer_pov == "Stone")
+    return "you win!";
+  else if (mypov == "Reversed Paper" && computer_pov == "Stone")
+    return "you lose!";
+  else if (mypov == "Reversed Paper" && computer_pov == "Scissors")
+    return "you win!";
+  else if (mypov == "Reversed Stone" && computer_pov == "Scissors")
+    return "you lose!";
+  else if (mypov == "Reversed Stone" && computer_pov == "Paper")
+    return "you win!";
+};
+
+var counting_message = function (win_count, loss_count, name_input) {
+  //counting winning
+  return (
+    "Hi " +
+    name_input +
+    "  so far you've been winning" +
+    "</br>" +
+    win_count +
+    "/" +
+    loss_count +
+    "<br>  turns"
+  );
+};
```

Comment:
Try using template literals: 
```suggestion
var counting_message = function (win_count, loss_count, name_input) {
  //counting winning
  return `Hi ${name_input}, so far you have been winning ${win_count} / ${loss_count} games!`;
};
```